public/index.html

```diff
@@ -0,0 +1,370 @@
+            <li class="flex-item-1">
+                <h2>UI &#38; UX Design</h2>
+                <img src="./imgs/brush.gif" alt="brush logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>LABORATORY</h2>
+                <img src="./imgs/erhlenmeyer.gif" alt="flask logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>DEVELOPMENT</h2>
+                <img src="./imgs/gear.gif" alt="gear logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+        </ul>
+        <h2 class="subheader">PRICING TABLES</h2>
+        <section class="table-section flex-container">
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>BASIC</th>
+                    </tr>
+                    <tbody>
+                        <tr>
+                            <td class="table-price">$5.99/MONTH</td>
+                        </tr>
+                        <tr>
+                            <td>3 Projects</td>
+                        </tr>
+                        <tr>
+                            <td>2 Users</td>
+                        </tr>
+                        <tr>
+                            <td>1 Gb user</td>
+                        </tr>
+                        <tr>
+                            <td>1 Calendar </td>
+                        </tr>
+                        <tr class="table-foot">
+                            <td>
+                                <button type="button" class="button-signup">SIGN UP</button>
+                            </td>
+                        </tr>
+                    </tbody>
+            </table>
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>STANDART</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="table-price">$10.99/MONTH</td>
+                    </tr>
+                    <tr>
+                        <td>7 Projects</td>
+                    </tr>
+                    <tr>
+                        <td>5 Users</td>
+                    </tr>
+                    <tr>
+                        <td>10 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>5 Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
```

Comment:
Not that I'm advocating its use here, but since you're already using the other table structures, there's `tfoot`, which would be more semantically accurate.

However, as I mentioned above, you don't need to use the table structures for something like this; I would rather see something like `tr:last-child` as the selector for this row.